cli/js/files.ts

```diff
@@ -180,7 +193,10 @@ export function writeSync(rid: number, p: Uint8Array): number {
  *
  */
 export async function write(rid: number, p: Uint8Array): Promise<number> {
-  const result = await sendAsyncMinimal(dispatch.OP_WRITE, rid, p);
+  if (OP_WRITE < 0) {
+    OP_WRITE = OPS_CACHE["op_write"];
+  }
```

Comment:
I still suggest to test this path using regular lookup, but that can be done as an experiment in separate PR